src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -1,12 +1,29 @@
-import React, { MouseEventHandler, useCallback } from "react";
+import React, {
+  RefObject,
+  MouseEventHandler,
+  useCallback,
+  useRef,
+} from "react";
 import classNames from "classnames";
 import { useCss } from "react-use";
+import { minutesToHours } from "date-fns";
```

Comment:
I checked out the branch and I get an error here.
`Module '"date-fns"' has no exported member 'minutesToHours'`

I checked `date-fns` but I don't see this function being exported, actually it doesn't exist at all.

Reply:
did you run npm i recently? we've updated date-fns to latest version a while ago @denisdimitrov

Reply:
merged https://github.com/sparkletown/sparkle/pull/1836/files